src/models/ActivityUpdateModel.ts

```diff
@@ -0,0 +1,8 @@
+import Joi from 'joi';
+import { WaitReason } from '@dvsa/cvs-type-definitions/types/v1/enums/waitReason.enum';
+
+export const ActivityUpdateModel = Joi.object().keys({
```

Comment:
Same here, why has this been re-named to be activity model, it isn't a model it's a validation schema?